docs/source/clustering/dunn_index.rst

```diff
@@ -0,0 +1,21 @@
+.. customcarditem::
+   :header: Dunn Index
+   :image: https://pl-flash-data.s3.amazonaws.com/assets/thumbnails/clustering.svg
```

Comment:
```suggestion
   :image: https://pl-flash-data.s3.amazonaws.com/assets/thumbnails/default.svg
```
the `clustering.svg` does not exist, we create one in the future and change them